### DISCUSSION TITLE
Events tagged with levels of triggering motion

### DISCUSSION THREAD

**Discussion by jhj1974** [blakeblackshear/frigate#10517] [Ideas]
Currently experiencing intermittent false positives maybe 5, 10, 30 or more minutes apart. Intensely watching the debug view for motion boxes for prolonged times feels exhausting, and watching the clips really do not give me any clues to what actually triggered the event.

Yes, there is a car or sometimes two in the image (one is a tractor actually), but despite they have been stationary for hours they are repeatidly re-detected. Motion masks has been applied to areas of no interest, and no visible motion in the clips. Currently configured a threshold of 150 and a contour_area of 50,  wich feels maybe a little to high.

Would it be possible to tag each event with the (peak) level of motion and the area of grouped pixels that triggered the event to assist in tuning motion detection? Maybe also information about how large a portion of the image experienced substantial changes as to aid tuning lightning_threshold?

![image](https://github.com/blakeblackshear/frigate/assets/96663288/a6424f52-6c2f-4654-906f-05e084111870)

**Comment by NickM-27** [comment]
as part of the UI rewrite, more information will be available about how much motion was detected at different points
